docs/README.md

```diff
@@ -0,0 +1,250 @@
+ ---> Using cache
+ ---> 5781d3cf82fd
+Step 4/7 : ENV TERM=xterm-256color APP=/opt/app
+ ---> Using cache
+ ---> 6b1a298cc237
+Step 5/7 : ADD requirements.txt $APP/
+ ---> Using cache
+ ---> 9f17f48879c1
+Step 6/7 : RUN cd $APP && pip install -r requirements.txt
+ ---> Using cache
+ ---> 52cfb162c7f2
+Step 7/7 : ADD . $APP
+ ---> ad287ad93ed2
+Successfully built ad287ad93ed2
+Successfully tagged registry.k8s.local/teracy_teracy-dev-docs-dev:a2115e0-dirty
+The push refers to repository [registry.k8s.local/teracy_teracy-dev-docs-dev]
+3d9bbc3b0aa7: Preparing
+a3443c79712d: Preparing
+ad4d80495365: Preparing
+6ee5cb4eed32: Preparing
+bb839e9783c7: Preparing
+237ce60325c6: Preparing
+1b976700da1f: Preparing
+bde41e1d0643: Preparing
+237ce60325c6: Waiting
+1b976700da1f: Waiting
+7de462056991: Preparing
+3443d6cf0f1f: Preparing
+f3a38968d075: Preparing
+a327787b3c73: Preparing
+5bb0785f2eee: Preparing
+bde41e1d0643: Waiting
+7de462056991: Waiting
+3443d6cf0f1f: Waiting
+f3a38968d075: Waiting
+a327787b3c73: Waiting
+5bb0785f2eee: Waiting
+a3443c79712d: Layer already exists
+6ee5cb4eed32: Layer already exists
+bb839e9783c7: Layer already exists
+ad4d80495365: Layer already exists
+237ce60325c6: Layer already exists
+1b976700da1f: Layer already exists
+bde41e1d0643: Layer already exists
+7de462056991: Layer already exists
+3443d6cf0f1f: Layer already exists
+f3a38968d075: Layer already exists
+a327787b3c73: Layer already exists
+5bb0785f2eee: Layer already exists
+3d9bbc3b0aa7: Pushed
+a2115e0-dirty: digest: sha256:b819729f0229ae62e0297f0d62b4a65db04495ee0e82cb0db1e1f547c5c29c5b size: 3055
+Build complete in 32.784044601s
+Starting test...
+Test complete in 16.52µs
+Starting deploy...
+Error: release: "teracy-dev-docs-dev" not found
+Helm release teracy-dev-docs-dev not installed. Installing...
+No requirements found in helm-charts/teracy-dev-docs-dev/charts.
+NAME:   teracy-dev-docs-dev
+LAST DEPLOYED: Wed Mar  6 10:35:46 2019
+NAMESPACE: teracy-dev
+STATUS: DEPLOYED
+
+RESOURCES:
+==> v1/Service
+NAME                 TYPE       CLUSTER-IP     EXTERNAL-IP  PORT(S)  AGE
+teracy-dev-docs-dev  ClusterIP  10.233.36.154  <none>       80/TCP   0s
+
+==> v1beta2/Deployment
+NAME                 DESIRED  CURRENT  UP-TO-DATE  AVAILABLE  AGE
+teracy-dev-docs-dev  1        1        1           0          0s
+
+==> v1beta1/Ingress
+NAME                 HOSTS                      ADDRESS  PORTS  AGE
+teracy-dev-docs-dev  dev.docs.teracy-dev.local  80, 443  0s
+
+==> v1/Pod(related)
+NAME                                  READY  STATUS             RESTARTS  AGE
+teracy-dev-docs-dev-5ccfc8b869-2l7mk  0/1    ContainerCreating  0         0s
+
+
+NOTES:
+1. Get the application URL by running these commands:
+  https://dev.docs.teracy-dev.local/
+
+Deploy complete in 1.669862476s
+Watching for changes every 1s...
+```
+
+- Configure the domain alias by editing the `k8s-dev/workspace/teracy-dev-entry/config_override.yaml`
+with the following configuration:
```

Comment:
the following configuration --> the configuration below:

("following" is used to many times :))